storage/migrations/25_gasoutputs_height_code.go

```diff
@@ -0,0 +1,22 @@
+package migrations
+
+import "github.com/go-pg/migrations/v8"
+
+// Schema version 22 adds Height and ActorName to gas outputs table
+
+func init() {
+	up := batch(`
+	ALTER TABLE public.derived_gas_outputs ADD COLUMN height bigint NOT NULL;
+	ALTER TABLE public.derived_gas_outputs ADD COLUMN actor_name text NOT NULL;
+	ALTER TABLE public.derived_gas_outputs DROP CONSTRAINT derived_gas_outputs_pkey;
```

Comment:
I also see in other migrations we do `_pk` instead of `_pkey`, should I change that?

Reply:
pkey is the convention used by postgres when you create a primary key